FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,17 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else {
+				return
+			}
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data, response))
+			case .failure(_):
```

Comment:
Since we are ignoring the specific `Error` instance, we can omit the attribution completely:
```suggestion
			case .failure:
```